src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfile.java

```diff
@@ -17,6 +17,7 @@
 
 import com.autotune.analyzer.kruizeObject.SloInfo;
 import com.autotune.analyzer.recommendations.term.Terms;
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
Before this pull request, as we discussed earlier, we need a straightforward PR that simply changes the name from PerformanceProfile to MetricProfile without altering the functionality. This current PR should build upon that.